sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -1481,9 +1481,9 @@ def __init__(
         :rtype: dict(str, Any)
         """
         options = self._upload_page_options(
-            page,
-            start_range,
-            end_range,
+            page=page,
```

Comment:
out of curiosity...what is the error pylint throws here?

Reply:
Specify parameter names when making calls with more than 2 positional parameters.